sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>1.8.0</version>
```

Comment:
For 1, I'll create another PR since maven build seems to be unstable for me. It fails due to timeout frequently during this PR.